holmes/core/conversations_worker/realtime_manager.py

```
@@ -313,109 +310,33 @@ async def _run(self) -> None:
-            while not self._stop_event.is_set():
-                now = asyncio.get_running_loop().time()
-
-                # Detect channel closure or silently-dead WS. The library's
-                # auto-reconnect is unreliable on clean closes, so we do our
-                # own full teardown/reconnect on any failure signal.
-                unhealthy_reason = self._channel_unhealthy()
-                if unhealthy_reason is not None:
+                    await self._full_reconnect()
+                    attempt = 0
+                    await self._serve()
+                except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)